README.md

```diff
@@ -67,3 +82,11 @@ machine. `-d` option sets the disk and `-p` for the port and baudrate.
 ```
 python3 test_psa_target.py -t GNUARM -m ARM_MUSCA_B1 -d D: -p COM8:115200
 ```
+
+Note: Vagrant build environment will not be able to access target via USB, so
+only use `-b` option to build.
```

Comment:
I'd suggest to move this instruction to vagrant/README with detailed instructions on how to use vagrant. This main README can focus on running the tests in a Linux environment

Reply:
`Vagrant/README.md` doesn't specify the use of `test_psa_target.py` command, therefore its been included here.

Reply:
Vagrant is used for building so building instructions should be in Vagrant/README.md.
Assuming that the binary has been built, what is the instruction to run all the tests? 
If you also add this information then it's fine

Reply:
At the moment there no way you can run those tests from a Vagrant machine to a target directly, as the USB is not shared by the host machine. All you can do is copy the binary from `/vagrant` to host machine and manually run. In the case of ` test_psa_target.py`, it only builds binaries and doesn't store them separately for each test suite as it builds on top of each other. This issue will be addressed when Jenkins CI is going to be implemented to tests these targets.

The true purpose of `test_psa_target.py` is to build and execute all the tests one by one for the target which is connected to the host.

There is a note in `vagrant/README.md`
```Note that by default vagrant shares `/vagrant` in the virtual machine with the host. You can use this directory to copy binaries from inside the vagrant machine out to the host for e.g. programming via USB.```

I'm not sure what more information is needed here, please could you suggest me with an example?

Reply:
"All you can do is copy the binary from /vagrant to host machine and manually run"
Which command do you use to test only (assuming the tests have already been built using "-b")?

Reply:
> "All you can do is copy the binary from /vagrant to host machine and manually run"
> Which command do you use to test only (assuming the tests have already been built using "-b")?

You can't, because:

```
it only builds binaries and doesn't store them separately for each test suite as it builds on top of each other. This issue will be addressed when Jenkins CI is going to be implemented to tests these targets.
```

Reply:
Thanks for the explanation. So, what is missing is to say something like: 

"`test_psa_target.py` builds and executes all the tests one by one. The binaries are not stored separately for each test suite; each test suite is built on top of the previous one. This script cannot be executed in the Vagrant environment because the Vagrant machine does not have access to the USB of the host machine and therefore cannot run the tests. In the Vagrant environment, you can check that all tests can be built by using the `-b` option."